e2e/test/scenarios/dashboard-filters/parameters.cy.spec.js

```diff
@@ -592,6 +598,301 @@ describe("scenarios > dashboard > parameters", () => {
+        });
+
+        selectDashboardFilter(getDashboardCard(0), "Name");
+
+        getDashboardCard(0).within(() => {
+          cy.findByText("User.Name").should("exist");
+        });
+
+        getDashboardCard(1).within(() => {
+          cy.findByLabelText("close icon").click();
+        });
+
+        selectDashboardFilter(getDashboardCard(1), "Address");
+
+        getDashboardCard(0).within(() => {
+          cy.findByText("User.Name").should("exist");
+        });
+
+        getDashboardCard(1).within(() => {
+          cy.findByText("User.Address").should("exist");
+        });
+      });
+
+      it("should not automatically wire parameters to cards that don't have a matching field", () => {
+        cy.createQuestion({
+          name: "Products Table",
+          query: { "source-table": PRODUCTS_ID, limit: 1 },
+        }).then(({ body: { id: questionId } }) => {
+          createDashboardWithCards([
+            {
+              card_id: ORDERS_BY_YEAR_QUESTION_ID,
+              row: 0,
+              col: 0,
+              size_x: 5,
+              size_y: 4,
+            },
+            {
+              card_id: questionId,
+              row: 0,
+              col: 4,
+              size_x: 5,
+              size_y: 4,
+            },
+          ]).then(dashboardId => {
+            visitDashboard(dashboardId);
+            cy.wait("@dashboard");
+          });
+        });
+
+        cy.icon("pencil").click();
+
+        cy.findByTestId("dashboard-header").icon("filter").click();
+        popover().within(() => {
+          cy.contains("Text or Category").click();
+          cy.findByText("Is").click();
+        });
+
+        selectDashboardFilter(getDashboardCard(0), "Name");
+
+        getDashboardCard(0).within(() => {
+          cy.findByText("User.Name").should("exist");
+        });
+
+        getDashboardCard(1).within(() => {
+          cy.findByText("Select…").should("exist");
+        });
+      });
+
+      it("should autowire parameters to cards in different tabs", () => {
+        const cards = [
+          {
+            card_id: ORDERS_BY_YEAR_QUESTION_ID,
+            row: 0,
+            col: 0,
+            size_x: 5,
+            size_y: 4,
+          },
+          {
+            card_id: ORDERS_COUNT_QUESTION_ID,
+            row: 0,
+            col: 4,
+            size_x: 5,
+            size_y: 4,
+          },
+        ];
+
+        createDashboardWithCards(cards).then(dashboardId => {
+          visitDashboardAndCreateTab({
+            dashboardId,
+            save: false,
+          });
+          cy.wait("@dashboard");
+        });
+
+        cy.findByTestId("dashboard-header").icon("filter").click();
+        popover().within(() => {
+          cy.contains("Text or Category").click();
+          cy.findByText("Is").click();
+        });
+
+        cy.findByTestId("dashboard-header").icon("add").click();
+
+        cy.findByTestId("add-card-sidebar").findByText("Orders Model").click();
```

Comment:
Nit: we can maybe make a helper for this (if one doesn't already exist) since repeat it 3 times